ta-sdk-spi/src/main/java/com/ibm/ta/sdk/spi/util/Util.java

```diff
@@ -34,14 +37,38 @@ public static String getSDKVersion() {
     }
     return version;
   }
+
+  public static void zipCollection(Path zipOutFile, File zipInDir, boolean excludeData) throws IOException  {
+    ZipOutputStream zos = new ZipOutputStream(new FileOutputStream(zipOutFile.toFile()));
+    zos.putNextEntry(new ZipEntry(zipInDir.getName() + ZIP_FILE_SEPARATOR));
+    zos.closeEntry();
+
+    // Add subdir files
+    String parentDir = zipInDir.getName();
+    for (File dirFile : zipInDir.listFiles()) {
+      if (excludeData) {
+        // If plugin has sensitive data, include only the metadata.assessmentunit.json and reports HTML files in
+        // each assessment unit dir
+        addZipEntry(dirFile, parentDir, zos, Arrays.asList(new String[]{
+                TADataCollector.ASSESSMENTUNIT_META_JSON_FILE,
+                "recommendations_.*.html"}));
```

Comment:
The html reports are not required in the archive
but migration artifacts could be depending on how we intend to consume plugins within TA.

Reply:
Right, html reports are not required for import into TA. I've included it because it's a human readable version of the recommendations.json. If user might want to copy and view the reports on another system, they could just copy the zip. Or months after exporting the zip, user doesn't remember what the archive is for, they could look at the reports. 

So we could decide to remove it, but also harmless to include it as well. Please let me know...

Reply:
Good point.  Probably can leave them in for that reason.  I think what we need to do in future is switch to declaring what files need to be persisted in order to do a reassessment later - but not allowing them to be persisted if they contain confidential data (effectively disabling the reassessment capability).

Reply:
I think we can leave the inclusion of migration files to a separate commit for 0.6 but this is good to go for 0.5.x